Automattic/wp-calypso
# Checkout: Improve pending page UI

#### Proposed Changes

After completing a purchase in checkout, the user will be directed briefly (sometimes so briefly it's invisible!) to a "pending" page which directs the user to wait while their order completes. This page could use some visual improvements, which thie PR attempts to make.

Fixes https://github.com/Automattic/wp-calypso/issues/65941

This is part of implementing https://github.com/Automattic/wp-calypso/issues/53356

Before:

<img width="1512" alt="pending-current" src="https://user-images.githubusercontent.com/2036909/182001658-96c60485-1033-4b9e-8726-d8103e373f9b.png">

After (note that the dots are animated):

<img width="1512" alt="Screen Shot 2022-08-01 at 3 47 47 PM" src="https://user-images.githubusercontent.com/2036909/182232826-f8c7a5c1-62c4-406f-a9bb-db42283bf208.png">

#### Testing Instructions

Visit `/checkout/thank-you/no-site/pending/12345` and you will see the page briefly before a redirect.
